pyqmc/optimize_ortho.py

```diff
@@ -519,6 +519,19 @@ def optimize_orthogonal(
         correlated_options["client"] = client
         correlated_options["npartitions"] = npartitions
 
+    # warm up to equilibrate the configurations before running optimization
+    vmcoptions = {}
+ 
+    data, coords = pyqmc.mc.vmc(
+        wfs[-1],
+        coords,
+        accumulators={},
+        client=client,
+        npartitions=npartitions,
+        **vmcoptions
```

Comment:
Why do you set vmcoptions to be an empty dictionary and then pass it? I guess we want a warmup_options=None as a parameter, and then set it equal to whatever defaults we want if it is None.